galaxy_update_configs_playbook.yml

```diff
@@ -0,0 +1,59 @@
+- hosts: galaxy-handlers
+  become: true
+  vars_files:
+      - group_vars/all.yml
+      - group_vars/galaxy_etca.yml
+      - group_vars/VAULT
+      - group_vars/galaxyservers.yml
+      - group_vars/galaxy_etca_slurm.yml
+      - host_vars/galaxy.usegalaxy.org.au.yml
+      - host_vars/galaxy-handlers.usegalaxy.org.au.yml
+      - secret_group_vars/stats_server_vault
+  handlers:
+    - include: handlers/galaxy_handlers.yml
+  tasks:
+    - name: copy job_conf file
+      template:
+        src: "{{ galaxy_config_template_src_dir }}/config/aarnet_job_conf.yml.j2"
```

Comment:
shouldn't this be galaxy_job_conf.yml.j2 ?

Reply:
yes!